source/features/lines-changed-per-commit.tsx

```diff
@@ -0,0 +1,53 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+
+const getCommitChanges = async (commits: string[]): Promise<string[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+				... on Commit {
+                    additions
+                    deletions
+					}
+				}
+			`).join('\n')}
+		}
+	`);
+
+	return repository;
```

Comment:
Again this function should return a raw API response, I repeatedly explained this in previous PRs.

Reply:
@fregante explain what you mean? I need 2 items from the response.  The commit and the additions / deletions. What do you expect the response to be?

Reply:
https://github.com/sindresorhus/refined-github/pull/2832#discussion_r384950532

Reply:
It returns an object like: 

```js
{
  _1a2b3c: {
    additions: 1,
    deletions: 1
  }
}
``` 
but should be

```js
{
  1a2b3c: {
    additions: 1,
    deletions: 1
  }
}
``` 

but yeah I guess in this case the difference isn’t much to justify the extra code. Plus what changes required for this PR skips the problem altogether